pyproject.toml

```diff
@@ -42,8 +42,8 @@ ignore-module = false
 ignore-nested-functions = false
 ignore-nested-classes = true
 ignore-setters = false
-fail-under = 62.5
-exclude = ["setup.py", "docs","tests","examples"]
+fail-under = 68.5
+exclude = ["setup.py", "docs","tests","examples","mava/lib"]
```

Comment:
Are you sure that this folder is not local to your PC? It does not exist on my side when I use docker or run locally with venv

Reply:
The folder is local for me, but it's one which can be created when building mava, so it's good to include it here to prevent local pre-commit checks from failing